spec/services/anonymise_user_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe AnonymiseUser do
+  let(:user) { create(:administrator) }
+
+  describe "#call" do
+    it "returns a successful result" do
+      result = described_class.new(user: user).call
+
+      expect(result.success?).to be(true)
+      expect(result.failure?).to be(false)
+    end
+
+    it "anonymises the email address" do
+      email = user.email
+      described_class.new(user:).call
```

Comment:
I tend to use `described_class` in shared examples or when the `ClassName::Is::ExceptionallyLong::AndWouldPushEverythingOffThePage`. I think:

- specs tend to be improved for getting the point home by repeating the `DoThingToThing` class name
- this is not a hill I would die on, `described_class` is perfectly acceptable, and it's a thing I like to discuss in passing